tests/regression/test_function_spaces.py

```diff
@@ -126,3 +131,105 @@ def test_VV_ne_VVV():
 
 def test_function_space_dir(cg1):
     dir(cg1)
+
+
+@pytest.mark.parametrize("meshes", [
+    (UnitIntervalMesh(1), UnitIntervalMesh(2)),
```

Comment:
Is this standard? This seems like we would create and hold all of these meshes in memory for the test suite duration. Might it be better to use something like `partial(UnitIntervalMesh, 1)` here instead? I honestly don't know.

Reply:
Yeah. I think the tests probably do this in various places, but you should use a `@pytest.fixture` for this to avoid constructing the meshes when collecting the tests.